web-jehlomat/src/routes.ts

```diff
@@ -64,6 +65,11 @@ export const routes: Route[] = [
         Component: Prihlaseni,
         path: '/prihlaseni',
     },
+    {
+        id: Routes.USER_NEW,
+        Component: SetNewPassword,
```

Comment:
Looks like this project has a convention to name Screen Components in Czech

Reply:
Jo i ne. Některé komponenty jsou v AJ (LandingPage, About, Welcome, ErrorPage a zbytek v CS); osobně dávám přednost AJ, ale pokud bude zájem můžu přejmenovat.

Reply:
Yes and no. Some components are using EN naming (like LandingPage, Welcome, About, ErrorPage, and the rest is in CS). But if there is preferred way, just tell me, I don't mind rename it (however it's gonna be a little bit mess (screen components in CS and visual components in EN :)

Reply:
Agree. I would love to have everything in English